pentaho-js/src/test/javascript/cdf/components/TableComponent-spec.js

```diff
@@ -196,5 +196,27 @@ define([
       dashboard.update(tableComponent);
     });
 
+    /**
+     * ## The Table Component # can update when server side is true
+     */
+    it("can update when server side is true", function(done) {
```

Comment:
This test feels far more like a functional test than a unit test, since you have so many "callThroughs" and binding to the "cdf:postExecution". You should be making more use of the mock objects to constrain the bounds of the test.